example/example.swift

```diff
@@ -133,6 +133,23 @@ let b = 50 /= 20
 ||=
 ~=
 
+// Custom Operators
+infix operator ～ {
+    precedence 20      // Define precedence
+    associativity none // Define associativity
+}
+
+func ～(n: Int, w: Int) -> String {
```

Comment:
Looks like we can remove this example since there's nothing else new here? We want to keep this as small as possible so it's reasonable to skim for regressions.